packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -107,6 +130,18 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
           <span className="inline-block" ref={ancientWord}>
             {word.text}
           </span>
+          <Button
+            className={
+              hasFootnote || (hasTranslatorNote && hasLanguageReadPermissions)
```

Comment:
@Pertempto I think I did mess up the order there. Sorry about that.

Reply:
We also want to check the write permission, not the read permission, correct?

Reply:
Now that I re-read Adrian's logic, I think maybe your logic was correct, it just needed to check read permission. I'm getting myself confused now 😆 

I think it would make things more clear if you renamed `hasLanguageReadPermissions` to `isTranslator`

After that, I think we'd write the logic like this:


`hasFootnote || (hasTranslatorNote && isTranslator)`

Anyone can view the foot notes, only translators can view the translator note

Reply:
@arrocke please correct me if I'm wrong again.

Reply:
Yes this is the right idea. We don't want to expose translator notes to users who can't view them.